azext_iot/deviceupdate/commands_account.py

```diff
@@ -0,0 +1,223 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from azext_iot.common.utility import handle_service_exception
+from azext_iot.deviceupdate.common import (
+    ADUPublicNetworkAccessType,
+    ADUAccountSKUType
+)
+from azext_iot.deviceupdate.providers.base import (
+    DeviceUpdateMgmtModels,
+    DeviceUpdateAccountManager,
+    parse_account_rg,
+    ARMPolling,
+)
+
+logger = get_logger(__name__)
+
+# Accounts
```

Comment:
these comments can be removed in the final pr since the file name is descriptive